openioc2stix/translate.py

```diff
@@ -129,5 +220,30 @@ def to_cybox(infile):
 
 
 def to_stix(infile):
-    """Converts the `infile` OpenIOC document into a STIX Package."""
-    pass
+    """Converts the `infile` OpenIOC xml document into a STIX Package.
+
+    Args:
+        infile: OpenIOC xml filename to translate
+
+    Returns:
+       STIX xml package ready to be output to a data stream.
```

Comment:
I think this makes it sounds like textual XML data is being returned, I'd go with "A `stix.core.STIXPackage` object"